### DISCUSSION TITLE
Do not work with multiple subscriptions

### DISCUSSION THREAD

**Discussion by PassHours** [PrateekKumarSingh/AzViz#69] [General]
Hi,

Just to inform that it do not work with multiple subscriptions. Just fails to identity the resource group and rightly so, as it do not know in which subscription to look for. Please fix. Thanks

PS C:\WINDOWS\system32> Export-AzViz -ResourceGroup recover-rg -Theme light -OutputFormat png -Show

    █████╗ ███████╗██╗   ██╗██╗███████╗
   ██╔══██╗╚══███╔╝██║   ██║██║╚══███╔╝   Author    : Prateek Singh (Twitter @singhprateik)
   ███████║  ███╔╝ ██║   ██║██║  ███╔╝    Module    : Azure Visualizer v1.2.1
   ██╔══██║ ███╔╝  ╚██╗ ██╔╝██║ ███╔╝     Github    : https://github.com/PrateekKumarSingh/AzViz
   ██║  ██║███████╗ ╚████╔╝ ██║███████╗   Document  : https://azviz.readthedocs.io
   ╚═╝  ╚═╝╚══════╝  ╚═══╝  ╚═╝╚══════╝
 ▶ Testing Graphviz installation... 278ms
    ▶ GraphViz installation path : C:\Program Files\Graphviz\bin\dot.exe
 ▶ Configuring Defaults... 294ms
    ▶  Target Type            : Azure Resource Group
    ▶  Output Format          : png
    ▶  Exluded Resource Types :
    ▶  Output File Path       : C:\Users\AtanuGupta\AppData\Local\Temp\output.png
    ▶  Label Verbosity        : 1
    ▶  Category Depth         : 1
    ▶  Sub-graph Direction    : top-to-bottom
    ▶  Theme                  : light
    ▶  Launch Visualization   : True
 ▶ Target Azure Resource Groups...  329ms
    ▶ recover-rg
 ▶ Starting to generate Azure visualization... 334ms
    ▶ Exporting network associations for resource group: 'recover-rg'
Get-AzResourceGroup : 08:33:04 - Provided resource group does not exist.
At C:\Users\AtanuGupta\Documents\WindowsPowerShell\Modules\AzViz\1.2.1\src\private\ConvertFrom-Network.ps1:51 char:33
+ ... $location = Get-AzResourceGroup -Name $ResourceGroup -Verbose:$false  ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-AzResourceGroup], Exception
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.GetAzureResourceGroupCmdlet

**Comment by russrenpsg** [comment]
Try setting your subscription first with `Select-AzSubscription %subscription_name%` then rerun your command.
